README.md

```diff
@@ -14,7 +14,7 @@
 
 1. [Features](#features)
 2. [Disclaimer](#disclaimer)
-3. [Quark and remote PC](#quark-and-remote-pc)

```

Comment:
It'd be more coherent if the table of contents's name was also changed to "Quark and remote browsing"